.docs/user-guide/schedulers.md

```diff
@@ -159,15 +159,16 @@ configuring popular applications with persistent storage via Docker and REX-Ray.
 REX-Ray can be integrated with [Kubernetes](https://kubernetes.io/) allowing
 pods to consume data stored on volumes that are orchestrated by REX-Ray. Using
 Kubernetes' [FlexVolume](https://kubernetes.io/docs/user-guide/volumes/#flexvolume)
-plug-in, REX-Ray can provide uniform access to storage operatations such as attach,
+plug-in, REX-Ray can provide uniform access to storage operations such as attach,
```

Comment:
Shouldn't typo fixes and stuff like that be in a separate commit? It doesn't seems relevant for 1.6 support. :)